Comments/Models/Context.cs

```diff
@@ -113,6 +117,74 @@ public IEnumerable<Location> GetAllCommentsAndQuestionsForDocument(IList<string>
 			return sortedData;
 		}
 
+		/// <summary>
+		/// Organisation users view each others submitted responses to inform their own, but can't interact with or submit them.
+		/// </summary>
+		/// <param name="sourceURIs"></param>
+		/// <param name="partialMatchSourceURI">True if data is being retrieved for the review page</param>
+		/// <returns></returns>
+		public OrganisationCommentsAndQuestions GetOtherOrganisationUsersCommentsAndQuestionsForDocument(IList<string> sourceURIs)
+		{
+            var commentsAndAnswers = Location.IgnoreQueryFilters()
+                .IncludeFilter(l => l.Comment.Where(c => c.StatusId == (int)StatusName.SubmittedToLead
+                                                         && _organisationIDs.Any(o => o.Equals(c.OrganisationId))
+                                                         && !_organisationUserIDs.Contains((int)c.OrganisationUserId)))
```

Comment:
it really doesn't matter a lot, but instead of casting the nullable int to an int, you could just do:

&& !_organisationUserIDs.Contains(c.OrganisationUserId.Value)))

which i think is a little nicer. it's no big deal though.